lib/FundingProgrammes.php

```diff
@@ -30,6 +30,7 @@ public function transform(Entry $entry)
         $commonFields = ContentHelpers::getCommonFields($entry, $status, $this->locale);
 
         return array_merge($commonFields, [
+            'isArchived' => $commonFields['status'] === 'expired' && $entry->legacyPath !== null,
```

Comment:
Do we need that second half (the `legacyPath` bit)? I think some of the non-legacy programmes have already expired (EYP?).